var/spack/repos/builtin/packages/grass/package.py

```diff
@@ -93,7 +94,7 @@ class Grass(AutotoolsPackage):
     depends_on('opencl', when='+opencl')
     depends_on('bzip2', when='+bzlib')
     depends_on('zstd', when='+zstd')
-    depends_on('gdal', when='+gdal')  # required?
+    depends_on('gdal@:3.2.3', when='+gdal') 
```

Comment:
```suggestion
    depends_on('gdal@:3.2.999', when='+gdal') 
```
In case a new patch release comes out for 3.2.4

Reply:
Oh interesting! Okay that makes sense.